Libraries/LibCrypto/Cipher/Mode/CTR.h

```diff
@@ -162,7 +162,7 @@ class CTR : public Mode<T> {
         // FIXME: We should have two of these encrypt/decrypt functions that
         //        we SFINAE out based on whether the Cipher mode needs an ivec
         ASSERT(ivec.has_value());
-        auto iv = ivec.value();
+        auto iv = ByteBuffer(ivec.value());
```

Comment:
stylistic decision, your choice:
`ByteBuffer iv = ivec.value()` would do the same

Reply:
I was defensive here, because we change the ivec later and I wasn't sure whether we're allowed to modify it.

I overlooked that we actually get a *copy* of the `Optional`, not a reference, so we're free to modify the ivec as we please. I'll change the code to save those precious nanoseconds :)